### ISSUE TITLE
Word.Confidence and Word.Text not loaded up

### ISSUE THREAD

**Issue by andboc** [charlesw/tesseract-ocr-dotnet#8]
Hi, first of all thank you for efforts on making this wrapper work on VS2010. It will be great if you implement a monitor in order to have Word.Confidence and Word.Text loaded after an AnalyzeLayout run.
Thank you

**Comment by andboc** [comment]
Ops, just found a possible solution on this, but haven't understood well how to implement that in your code. Please have a look to thread at http://code.google.com/p/tesseractdotnet/issues/detail?id=19#c1

**Comment by charlesw** [comment]
Yes this is a known issue that I've come across before, it's a bit misleading to have word text and confidence on the results of AnalyzeLayout given they aren't ever initialized due. I generally used AnalyzeLayout as part of a pre-processing stage such as working out OCR regions etc. I'm currently experimenting with of reworking the API to make this stuff easier and more evident in how you use it while also looking at 64bit support and supporting 3.02 (Issues #6 and #4 respectively).

**Comment by andboc** [comment]
Ok, you are right, not a good idea to have them loaded up by AnalyzeLayout. I saw there's a ResultIterator class ready made in the sources which can be used during Recognize phase, I'm experimenting on how to call it during recognition, but for now no luck on that.
